tools/dev_qa.py

```diff
@@ -76,9 +76,9 @@
                 --bes_backend=remote.buildbuddy.dev \
                 --bes_results_url=https://app.buildbuddy.dev/invocation/ \
                 --remote_timeout=10m \
-                --extra_execution_platforms=@buildbuddy_toolchain//:platform \
-                --host_platform=@buildbuddy_toolchain//:platform \
-                --platforms=@buildbuddy_toolchain//:platform \
+                --extra_execution_platforms=//platforms:local_config_platform \
+                --host_platform=//platforms:local_config_platform \
+                --platforms=//platforms:local_config_platform \
```

Comment:
same problem here, we need to rework some of our flags usages (i.e. removing crosstool_top in favor of extra_toolchain).

Could you please leave this hunk out of your PR? I could take a look separately.

Reply:
👍